streamwebs_frontend/streamwebs/templates/streamwebs/datasheets/water_quality_view.html

```diff
@@ -0,0 +1,264 @@
+{% extends "streamwebs/data_view.html" %}
+
+{% load staticfiles %}
+
+{% block title %}Water Quality - {{data.site.name}} - Streamwebs{% endblock %}
+
+{% block body_title %}<a href='{% url 'streamwebs:site' data.site.slug %}'>{{data.site.name}}</a>
+» Water Quality ({{data.date}}){% endblock %}
+
+{% block scripts %}
+<script type="application/javascript" src="{% static 'streamwebs/js/data_view.js' %}"></script>
+<script type="application/javascript" src="{% static 'streamwebs/js/data/view/water_quality.js' %}"></script>
+{% endblock %}
+
+{% block content %}
+<div id="water_quality" class="datasheet">
+    <table>
+        <tr>
+            <td class="form-cell">
+                <p class="field-label">Stream/Site name: {{data.site.name}}</p>
+                <label for="date" class="field-label">Date:</label>
+                <input readonly type="date" name="date" value="{{data.date}}" />
+            </td>
+            <td class="form-cell">
+                <label for="DEQ Data Quality Level" class="field-label">DEQ Data Quality Level:</label>
+                <select name="DEQ Data Quality Level" required disabled>
+                    <option value="Level A" {% if data.DEQ_wq_level == 'A' %}selected{% endif %}>Level A</option>
```

Comment:
For the same reason all of the other inputs are readonly/disabled, so that it can display the values to the user without being editable.